### DISCUSSION TITLE
2024-02-03力扣面试经典150题 | 阿聪的小破站

### DISCUSSION THREAD

**Discussion by giscus** [SmartDengC/jet5devil-index#11] [Q&A]
# 2024-02-03力扣面试经典150题 | 阿聪的小破站

https://wiki.dengcong.com.cn/article/6pu2aw64/

**Comment by SmartDengC** [comment]
工作日的话:对于每天的算法,尽量在早上8点之前,进行,或者是在中午12点之后,或者是晚上
非工作日的话:  时间自定义
力扣每周的周赛在周天早上.
